### DISCUSSION TITLE
臺灣2050淨零排放路徑

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS3073-C/Python-Community-GS3073-C#387] [科技新知]
科學證實氣候變遷造成的影響已經相當緊急，氣候議題引發國際高度重視，各國陸續提出「2050淨零排放」的宣示與行動。為呼應全球淨零趨勢，2021年4月22日世界地球日蔡總統宣示，2050淨零轉型是全世界的目標，也是臺灣的目標。

我國2050淨零排放路徑將會以「能源轉型」、「產業轉型」、「生活轉型」、「社會轉型」等四大轉型，及「科技研發」、「氣候法制」兩大治理基礎，輔以「十二項關鍵戰略」，就能源、產業、生活轉型政策預期增長的重要領域制定行動計畫，落實淨零轉型目標。
[國發會政策內容介紹](https://www.ndc.gov.tw/Content_List.aspx?n=FD76ECBAE77D9811)
![164861199954718_P16498104](https://user-images.githubusercontent.com/90574174/173196697-a06e4f1c-dff8-49e1-9024-3a64d07b3f3a.jpg)
圖片來源: 經濟日報

台灣第一次有長達30年的環境保護相關政策蠻感動的，但也要有企業緊密配合才會見效，還需再觀察

**Comment by luo910227** [comment]
我自己認為，此噱頭跟實際狀況差距仍大。例如能源轉型方面碳封存占很重要一部分，但是目前仍在測試階段。
